packages/@react-aria/breadcrumbs/docs/useBreadcrumbs.mdx

```diff
@@ -210,27 +208,44 @@ function Breadcrumbs(props) {
 }
 
 function BreadcrumbItem(props) {
-  // Same as in the previous example...
-///- begin collapse -///
   let {breadcrumbItemProps} = useBreadcrumbItem(props);
```

Comment:
Maybe something like this to avoid the `isHeading` prop to `useBreadcrumbItem` and to keep the `elementType` correct?

```suggestion
  let {breadcrumbItemProps} = useBreadcrumbItem({
     ...props,
     elementType: props.isHeading ? 'h3' : 'a'
  });
```

Reply:
Yep that was the approach I had at first where I had this check in useBreadcrumbItem:
```
  let isHeading = ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'].includes(elementType);
  let itemProps: HTMLAttributes<HTMLDivElement> = {};

  if (!isHeading) {
    itemProps = {...linkProps};
  }

  if (isCurrent) {
    itemProps = {...itemProps, 'aria-current': ariaCurrent || 'page'};
  }
```
I forget what issue made me switch to use isHeading, but I can't remember it and it seems to work just fine. I'll keep the above as `let {breadcrumbItemProps} = useBreadcrumbItem(props);` and instead change the cloneElement part in Breadcrumbs to 
```
React.cloneElement(child, {
    isCurrent: i === children.length - 1,
    elementType: i === children.length - 1 ? 'h3' : 'a'
 })
```
so that the example isn't passing `isHeading` to `useBreadcrumbItem` at all